makefile

```diff
@@ -16,6 +17,10 @@ help:
 .PHONY:install
 install: install_python_packages
 
+.PHONY:install-ci
+install-ci: install_prerequisites
+	uv pip install --system --requirement requirements/development.txt
```

Comment:
why not use a virtualenv for ci too?

Reply:
I rejected this idea to start with, but now that I've slept on it, I think it makes mroe sense to create the virtualenv in CI. I'll switch over to that.